mm2src/coins/eth/eth_wasm_tests.rs

```diff
@@ -45,6 +45,7 @@ async fn test_send() {
         required_confirmations: 1.into(),
         chain_id: None,
         logs_block_range: DEFAULT_LOGS_BLOCK_RANGE,
+        nonce_lock: Arc::new(AsyncMutex::new(())),
```

Comment:
You can use `new_nonce_lock` helper mentioned in another comment.

Reply:
done

Reply:
@laruh It's not used here still :)

Reply:
Done